x-pack/platform/plugins/shared/alerting/server/rules_client/lib/validate_actions.test.ts

```diff
@@ -314,19 +314,21 @@ describe('validateActions', () => {
     );
   });
 
-  it('should return error message if the action is an EDR sub-feature connector type', async () => {
-    getBulkMock.mockResolvedValueOnce([{ actionTypeId: 'test.edr', name: 'test name' }]);
+  it('should return error message if the action is an Endpoint Security sub-feature connector type', async () => {
+    getBulkMock.mockResolvedValueOnce([
+      { actionTypeId: 'test.endpointSecurity', name: 'test name' },
+    ]);
     listTypesMock.mockResolvedValueOnce([
       {
-        id: 'test.edr',
-        name: 'EDR connector type',
-        subFeatureType: 'edr',
+        id: 'test.endpointSecurity',
+        name: 'endpoint security connector type',
+        subFeature: 'endpointSecurity',
       },
     ]);
     await expect(
       validateActions(context as unknown as RulesClientContext, ruleType, data, false)
     ).rejects.toThrowErrorMatchingInlineSnapshot(
-      '"Failed to validate actions due to the following error: Invalid EDR connectors"'
+      '"Failed to validate actions due to the following error: IEndpoint security connectors cannot be used as alerting actions"'
```

Comment:
`s/IEndpoint/Endpoint/`

Reply:
Thanks! Should have been fixed in this commit, https://github.com/elastic/kibana/pull/203503/commits/66103c8f4795e8368589a8fe275b1de7b85b23bb

Reply:
yup, was looking at a stake copy ...